### PR TITLE
Update to 1.4.4

### PR DESCRIPTION
**Pull request by Pop000100**
- Update to 1.4.4's localization format.
- Remove Redundant code.
- Increment Major Version

### CODE REVIEW

File: Localization/ru-RU.hjson [hunk 1]
```diff
@@ -4,5 +4,31 @@ Mods: {
...
+				}
+
+				DisplayItemTooltips: {
+					// Label: Display Items Tooltips
+					// Tooltip: Whether or not mod origins are displayed in item tooltips.
+				}
+
+				DisplayTechnicalNames: {
+					// Label: Display Technical Names
+					// Tooltip: Whether or not technical names are displayed.
```
Comment (gardenappl) [inline Localization/ru-RU.hjson:29]:
```suggestion
		WorldTooltipsOn: "[WMITF] Подсказки для НИП/блоков ВКЛ"
		WorldTooltipsOff: "[WMITF] Подсказки для НИП/блоков ВЫКЛ"

		Keybinds: {
			"Technical Names.DisplayName": Показывать технические названия
			"Tile/NPC Mod Tooltip.DisplayName": Показывать моды для НИП/блоков
		}

		Configs: {
			Config: {
				DisplayName: Конфигурация

				DisplayWorldTooltips: {
					Label: Показывать подсказки вне инвентаря
					Tooltip: Позволяет увидеть мод, добавивший данный блок или НИПа.
				}

				DisplayItemTooltips: {
					Label: Показывать подсказки в инвентаре
					Tooltip: Позволяет увидеть мод, добавивший данный предмет.
				}

				DisplayTechnicalNames: {
					Label: Показывать технические названия
					Tooltip: Показывает значения Mod.Name, ModItem.Name и пр. вместо названия мода.
```

File: build.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿displayName = Which Mod Is This From? (WMITF)
 author = gardenapple
-version = 2.6.3
+version = 3.0.0
```
Comment (gardenappl) [inline build.txt:3]: Again, not sure why the major version change.

File: Localization/ru-RU.hjson (hunk 1 above)
Comment (gardenappl) [reply Localization/ru-RU.hjson:29]: Hmm looks like I can "commit suggestion" right from GitHub's UI. Might as well do that.

File: build.txt (hunk 2 above)
Comment (Pop000100) [reply build.txt:3]: Just wanted it to be distinct. I'll change the second number if that is bothersome. I am aware that it is SemVer.

**Pull request merged**